internetarchive/openlibrary
# remove the word beta from Solr Editions

Closes #8979


fix: remove word 'beta' from 'Solr Editions Beta' (as it is no longer in beta [link](https://github.com/internetarchive/openlibrary/issues/8971#issuecomment-2023379932))

### Technical

first commit updates the i18n value as the key itself had the _beta_ word   
second commit removes _beta_ from the actual translations 
### Testing

Running locally:
- Log in as admin
- Navigate to `http://localhost:8080/search?q=%22a%22&mode=everything&first_publish_year=2023&publisher_facet=Norton+%26+Company%2C+Incorporated%2C+W.+W.&sort=readinglog`
- Check Solr Edition text is rendering correctly after key change
- Check text does not have the _Beta_ word 
### Screenshot

Before:
![image](https://github.com/internetarchive/openlibrary/assets/102969687/71914995-222f-49b5-8c66-fd3b414ae49d)
After (tested locally with admin account):
![2024-03-28_0000](https://github.com/internetarchive/openlibrary/assets/102969687/87a302f0-7c17-4b56-b1b6-c03b3c103e94)

### Stakeholders
